ModularTegustation/tegu_items/representative/research/jcorp.dm

```diff
@@ -48,15 +48,29 @@ GLOBAL_LIST_EMPTY(jcorp_upgrades)
 	. = ..()
 	GLOB.jcorp_upgrades += "Gacha Chance 4"
 
-/datum/data/lc13research/well_gacha
-	research_name = "Wishing Well Output Rates"
-	research_desc = "According to some information L corp has told us, this well seems to rely tends to be unpredictable, and open to chance, so we can lock some of the bad outcomes from occuring as often."
+//Abnormality Mechanic Luck
+
+/datum/data/lc13research/abno_mechanic_luck
+	research_name = "Contained Abnormality Luck Upgrade"
+	research_desc = "L Corp has asked us about possibly helping agents get a better chance with some abnormalities, from not getting spiked drinks, to games or Russian Roulette. With some PE, we can make their requests possible."
+	cost = LOW_RESEARCH_PRICE
+	corp = J_CORP_REP
+
+/datum/data/lc13research/abno_mechanic_luck/ResearchEffect(obj/structure/representative_console/caller)
+	. = ..()
+	GLOB.jcorp_upgrades += "Abno Luck"
+
+/datum/data/lc13research/tool_abno_gacha
+	research_name = "Tool Abnormality Luck Upgrade"
+	research_desc = "According to some information L corp has told us, some of their 'Tool Abnormalities' tends to be unpredictable, and open to chance, so we can lock some of the bad outcomes from occuring as often."
 	cost = AVERAGE_RESEARCH_PRICE
 	corp = J_CORP_REP
 
-/datum/data/lc13research/well_gacha/ResearchEffect(obj/structure/representative_console/caller)
+/datum/data/lc13research/tool_abno_gacha/ResearchEffect(obj/structure/representative_console/caller)
 	. = ..()
-	GLOB.jcorp_upgrades += "Well Gacha"
+	GLOB.jcorp_upgrades += "Tool Gacha"
```

Comment:
Cursory Review: I looked up the value and it adds 25 to the success rate of gambling tools. I would ask Kirie if this is good or if the value should be lower/higher.
Effect i can see currently on tools.
- TreeSap has its explosion chance reduced by 25%. In numbers it reduces the 20% chance to 15% but 25% reduction makes it sound better.
- Mirror has a flat 5 stats added to its alteration. This has the side effect of making rerolling stat penalty reduced by half.
- Promise has its force multiplier 1 gamble increased from 50% to 75%, 1.3 multiplier is increased by 25% to 50%. This one may possibly need to be reduced by 10% so its not a 3/4 chance of success.
-